### PR TITLE
Trivial updates

### PR DESCRIPTION
**Pull request by mkykadir**
See each commit for details

### CODE REVIEW

**Comment by Geod24** [comment]
Can you add the motivation for the second commit ? I'm wondering whether `addEnrollment` belongs to the basic `Ledger`, as a mirror to `addTransaction`. Another approach would be to have a basic `Ledger` without any pool (if that makes sense), and a derived class with pools (given that for a client tracking the chain, a pool might not make sense).

**Comment by mkykadir** [comment]
`addEnrollment` of Ledger is only used through `TestLedger` in tests. Source code uses enrollment manager only.

**Review by linked0 — APPROVED** [review]
LGTM

**Comment by omerfirmak** [comment]
> `addEnrollment` of Ledger is only used through `TestLedger` in tests. Source code uses enrollment manager only.

`TestLedger` already exposes enrollment manager thru the `enrollment_manager` getter. We might as well remove addEnrollment entirely.

File: source/agora/consensus/Ledger.d [hunk 1]
```diff
@@ -1401,27 +1401,6 @@ public class Ledger
...
-            enroll = the enrollment data to add
-            pubkey = the public key of the enrollment
-            height = block height for enrollment
-
-        Returns:
-            true if the enrollment data has been added to the enrollment pool
-
-    ***************************************************************************/
-
-    public bool addEnrollment (in Enrollment enroll, in PublicKey pubkey,
```
Comment (omerfirmak) [inline source/agora/consensus/Ledger.d:1418]: Commit message should be updated as well, feel free to self merge after that.

**Review by omerfirmak — APPROVED** [review]

**Pull request merged**